tools/build/make.py

```diff
@@ -93,8 +95,11 @@ def bootstrap_bmake(source_root, objdir_prefix, libbsd_install_dir):
 
     # HACK around the deleted file bmake/missing/sys/cdefs.h
     if sys.platform.startswith("linux"):
-        env["CFLAGS"] = "-I{}/include/bsd -DLIBBSD_OVERLAY".format(libbsd_install_dir)
-        env["LDFLAGS"] = "-L{}/lib -lbsd".format(libbsd_install_dir)
+        if libbsd_install_dir is None:
+            debug("Compiling bmake on linux without bootstrapping libbsd")
```

Comment:
I chose to print a debug statement instead of exiting with an error because at some point you might decide to not compile libbsd on linux if already installed. I can chage it to a sys.exit() if you think it's better.